nilearn/decoding/tests/test_same_api.py

```diff
@@ -97,10 +97,10 @@ def test_lipschitz_constant_loss_logreg():
 
 
 def test_graph_net_and_tvl1_same_for_pure_l1(max_iter=100, decimal=2):
-    ###############################################################
+    #
```

Comment:
Oops! Indeed I had autopep8 0.9.1 from debian. Just installed 1.2. Thx for
spotting these weirdos!

On Thu, Jul 30, 2015 at 8:44 AM, Loïc Estève <notifications@github.com>
wrote:

> In nilearn/decoding/tests/test_same_api.py
> <https://github.com/nilearn/nilearn/pull/739#discussion_r35842783>:
>
> > @@ -97,10 +97,10 @@ def test_lipschitz_constant_loss_logreg():
> >
> >
> >  def test_graph_net_and_tvl1_same_for_pure_l1(max_iter=100, decimal=2):
> > -    ###############################################################
> > +    #
>
> bring back the line of hashes
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nilearn/nilearn/pull/739/files#r35842783>.
>



-- 
DED